flow360/component/flow360_params/solvers.py

```diff
@@ -423,7 +425,7 @@ class TransitionModelSolver(GenericFlowSolverSettings):
     model_type: Optional[Literal["AmplificationFactorTransport"]] = pd.Field(
         "AmplificationFactorTransport", alias="modelType", const=True
     )
-    linear_solver: Optional[LinearSolver] = pd.Field(alias="linearSolver", default=LinearSolver())
+    linear_solver: Optional[LinearSolver] = pd.Field(alias="linearSolver")
```

Comment:
why removed default?

Reply:
Added it back, deleting duplicate defaults is now handled as a schema postprocessing step